features/270-next-skycli/commands.md

```diff
@@ -2,15 +2,14 @@
 
 ### skycli auth login
 
-`skycli auth login [--email=EMAIL] [--admin-key=ADMIN_KEY]`
+`skycli auth login [--email=EMAIL]`
```

Comment:
For CI environment, let's say someone wants to use Travis to run `skycli deploy` for their applications whenever Git repo updated, how would authentication works??